OsmAnd-java/src/main/java/net/osmand/GPXUtilities.java

```diff
@@ -1657,8 +1660,10 @@ public int getNonEmptySegmentsCount() {
 		}
 
 		public QuadRect getRect() {
-			double left = 0, right = 0;
-			double top = 0, bottom = 0;
+			return getRect(0, 0, 0, 0);
+		}
+
+		public QuadRect getRect(double left, double bottom, double right, double top) {
```

Comment:
where 
public QuadRect getRect(double left, double bottom, double right, double top) { is used ???